program_socket_filter.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2019 Dropbox, Inc.
+// Full license can be found in the LICENSE file.
+
+package goebpf
+
+//#include "bpf_helpers.h"
+import "C"
+
+import (
+	"fmt"
+
+	"golang.org/x/sys/unix"
+)
+
+// SocketFilterResult is eBPF program return code enum
+type SocketFilterResult int
+
+const (
+	SocketFilterDeny  SocketFilterResult = C.SOCKET_FILTER_DENY
+	SocketFilterAllow SocketFilterResult = C.SOCKET_FILTER_ALLOW
+
+	SO_ATTACH_BPF    = 50
```

Comment:
please use CamelCase for constants

Reply:
`SO_ATTACH_BPF` is kernel constant I didn't use CamelCase intentionally to make it consistent with other `unix.` constants like:
```go
err := unix.SetsockoptInt(p.sockFd, unix.SOL_SOCKET, SO_ATTACH_BPF, p.GetFd())
```